lego/apps/events/models.py

```diff
@@ -692,17 +692,15 @@ def waiting_registrations(self):
     def waiting_registration_count(self):
         return self.waiting_registrations.count()
 
-    def set_groups_with_view_permission(self, groups_with_view_permission):
-        if len(groups_with_view_permission) > 0:
-            self.groups_with_view_permission.clear()
+    def set_can_view_groups(self, can_view_groups):
```

Comment:
I think this method can be removed now, since this only really sets `require_auth` :thinking:. (Might need it to set it still if there is no easier method though)